### PR TITLE
Add "accessibility_label" field to Button interface

### PR DESCRIPTION
**Pull request by kpeters-cbsi**
###  Summary

The `Button` interface in `@slack/types` is missing the `accessibility_label` field as described in the [API reference](https://api.slack.com/reference/block-kit/block-elements#button). This PR adds that field.

NB: while I have read the discussion in #1227 , I think this PR is still appropriate as the missing field is part of the published API and is not present in the Typescript definitions.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by filmaj — APPROVED** [review]
Thanks and LGTM! Would like another maintainer's approval, though, before merging, as I'm not proficient in TS 😛

**Review by seratch — APPROVED** [review]

**Pull request merged**
